products/opensuse/main.pm

```diff
@@ -654,19 +665,50 @@ sub load_extra_tests () {
     return 0;
 }
 
+sub load_otherDE_tests() {
```

Comment:
I am not talking about the testsuite but the test modules name, e.g. the rows with the colored background on the details page.